docs/articles_en/openvino-workflow/model-optimization.rst

```diff
@@ -13,57 +10,109 @@ Model Optimization Guide
+| :doc:`Training-time Optimization <model-optimization-guide/compressing-models-during-training>`:
+|      involves a suite of advanced methods such as Structured or Unstructured Pruning, as well
+       as Quantization-aware Training. This kind of optimization requires the use of the model's
+       original framework, for NNCF, it is either PyTorch or TensorFlow.
+
+| :doc:`Weight Compression <model-optimization-guide/weight-compression>`:
+|      an easy-to-use method for Large Language Model footprint reduction and inference
+       acceleration.
+
+
+To learn about the full scope of the framework, its installation, and technical details, visit
+both `the NNCF repository <https://github.com/openvinotoolkit/nncf?tab=readme-ov-file>`__ and
+`NNCF API documentation <https://openvinotoolkit.github.io/nncf/autoapi/nncf/>`__.
 
-Post-training Quantization is the fastest way to optimize an arbitrary DL model and should
-be applied first, but it is limited in terms of achievable accuracy-performance trade-off.
-The recommended approach to obtain OpenVINO quantized model is to convert
-`a model from original framework <https://huggingface.co/models>`__ to ``ov.Model``
-and ensure that the model works correctly in OpenVINO, for example, by calculating the
-model metrics. Then, ``ov.Model`` can be used as input for the ``nncf.quantize()``
-method to get the quantized model or as input for the ``nncf.compress_weights()``
-method to compress weights of Large Language Models (see the diagram below).
 
-In case of unsatisfactory accuracy or performance after Post-training Quantization,
-Training-time Optimization can be used as an option.
 
 .. image:: ../assets/images/DEVELOPMENT_FLOW_V3_crunch.svg
 
-Once the model is optimized using the aforementioned methods, it can be used for
-inference using the regular OpenVINO inference workflow. No changes to the inference
-code are required.
 
-.. image:: ../assets/images/WHAT_TO_USE.svg
+.. tab-set::
 
-Additional Resources
-####################
+   .. tab-item:: Installation
+      :sync: install
+
+      .. tab-set::
+
+         .. tab-item:: PyPI
+            :sync: pip
+
+            .. code-block::
+
+               pip install nncf
+
+         .. tab-item:: Conda
+            :sync: conda
+
+            .. code-block::
 
-- :doc:`Post-training Quantization <model-optimization-guide/quantizing-models-post-training>`
-- :doc:`Training-time Optimization <model-optimization-guide/compressing-models-during-training>`
-- :doc:`Weight Compression <model-optimization-guide/weight-compression>`
-- :doc:`Deployment optimization <running-inference/optimize-inference>`
-- `Hugging Face Optimum Intel <https://huggingface.co/docs/optimum/intel/optimization_ov>`__
+               conda install -c conda-forge nncf
+
+      For more installation details, see the page on
+      `NNCF Installation <https://github.com/openvinotoolkit/nncf/blob/develop/docs/Installation.md>`__.
+
+   .. tab-item:: System requirements
```

Comment:
These changes require regular maintaining and can lead to unaligned system requirements in the OpenVINO documentation and NNCF repository. I would suggest to have one place for store system requirements and link to it from OpenVINO documentation or not add it to OpenVINO documentation, because this information can be found in the NNCF repository or PyPi.

Reply:
Applied in https://github.com/openvinotoolkit/openvino/pull/25981/commits/72656191a194d7db5183939a0b992b07466d20fe